lib/gems/pending/VMwareWebService/MiqVimInventory.rb

```diff
@@ -64,11 +65,17 @@ def initialize(server, username, password, cacheScope = nil)
         deleteProperty(:HostSystem, "config.dateTimeInfo")
       end
     else
-      if @v4 && cacheScope != :cache_scope_event_monitor
+      if @v4
```

Comment:
@roliveri I just moved that conditional down [here](https://github.com/ManageIQ/manageiq-gems-pending/pull/24/files/7aab7da6d2624d0b984281c0b3b529d1233efc2d#diff-618aa8640f55186534c8e2be1b5b38b4R73l) so those two add/delete properties are the same but we're removing the datastore maintenance mode for all 4.x VCs